.github/workflows/build-and-test.yml

```diff
@@ -12,19 +12,49 @@ on:
       - epic/*
     paths:
       - '.github/workflows/build-and-test.yml'
+      - '.github/workflows/scripts'
       - '.github/actions'
       - '**.rs'
       - '**.toml'
+      - 'bindings/wasm/'
+      - '!bindings/wasm/**.md'
 
 env:
   RUST_BACKTRACE: full
+  CARGO_INCREMENTAL: 0 # disabled to reduce target cache size and improve sccache (https://github.com/mozilla/sccache#known-caveats)
   SCCACHE_CACHE_SIZE: 2G
   SCCACHE_IDLE_TIMEOUT: 0
   # SCCACHE_RECACHE: 1 # uncomment to clear sccache cache, then re-comment
 
 jobs:
+
+  determine-modification:
+    runs-on: ubuntu-latest
+    outputs:
+      core-modified: ${{ steps.change-detection.outputs.core-modified }} # map step output to job output
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run change detection
+        id: change-detection
+        run: |
+          echo comparing $(git rev-parse HEAD^) and $(git rev-parse HEAD)
+          #https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-emgitdiffemltoptionsgt--ltpathgt82308203
+          if [[ $(git diff HEAD^ HEAD -- $1) != '' ]]; then
```

Comment:
What is `$1` meant to be? Previously I think `':!bindings'` was passed to the script?

Reply:
good catch!